.github/workflows/tagging.yml

```diff
@@ -0,0 +1,43 @@
+name: Tagging release
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - master
+jobs:
+  tagging-release:
+    name: Tagging release
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged && github.head_ref == 'prerelease' && github.base_ref == 'master'
+    steps:
+      - uses: actions/checkout@v2
+      - name: get commit message
+        shell: bash
+        run: |
+          echo ::set-env name=commitmsg::$(git log --format=%B -n 1 ${{ github.event.after }})
+          echo $commitmsg
+      - name: get tag version
+        run: |
+          echo ::set-env name=tagversion::$( sed -e 's#.*: \(\)#\1#' <<< ${commitmsg} )
+          echo $tagversion
+      - name: set tag version
+        id: set_tag_version
+        run: |
+          git config --local user.email "one.amex@aexp.com"
+          git config --local user.name "OneAmexBot"
+          echo ${tagversion}
+          git tag ${tagversion}
+          git push origin --tag
+          echo ::set-output name=action_tag::${tagversion}
+      - name: Trigger Image Publishing
```

Comment:
Should we rely on either the tag being created/pushed or the `repository dispatch`? Would this trigger the `One App Docker Release` workflow twice?

Reply:
Fair point @nellyk did you notice if the docker-release workflow was triggered by this or by the fact a new tag was pushed? not really sure what we are dispatching here?

Reply:
Initially i wanted this to trigger the release, but unfortunately github actions does not support this. This wouldn't  tigger when a new tag was automatically created, that's why i added the `repository dispatch`

Reply:
automatically created tag - doesn't trigger the docker release workflow unless `repository dispatch` is used
tag pushed or created manually - triggers the docker release workflow

Reply:
Makes sense for git operations that occur inside a workflow do not trigger other workflows (to avoid cyclical runs). Thank you for confirming!

Reply:
Thanks @nellyk for clarifying, so @Francois-Esquire one will trigger for the manual tag push and this one will trigger for the automatic tag generated by this specific workflow. 👍